snuba/web/split.py

```diff
@@ -340,21 +326,11 @@ def execute(
                     exc_info=True,
                 )
 
-        minimal_ast_cols = {
+        minimal_columns = {
             (col.table_name, col.column_name)
             for col in minimal_query.get_all_ast_referenced_columns()
         }
-        minimal_ast_count = len(minimal_ast_cols)
-        minimal_legacy_cols = minimal_query.get_all_referenced_columns()
-        minimal_count = len(minimal_legacy_cols)
-        if minimal_count != minimal_ast_count:
-            logger.warning(
```

Comment:
This was a limitation of the legacy query. It was counting literals as columns. So the right result is the one from the AST